### PR TITLE
ci: require comment on exported `const`/`var`/`func`/`type`

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by Lou1415926 — APPROVED** [review]
My linter also complains (showing a waving line underneath the var) but sometimes I'm just too blind to see

File: internal/pkg/aws/apprunner/apprunner.go [hunk 1]
```diff
@@ -30,7 +30,8 @@ const (
 	// App Runner ImageRepositoryTypes
 	repositoryTypeECR       = "ECR"
 	repositoryTypeECRPublic = "ECR_PUBLIC"
-	// App Runner EndpointsID
+
+	// EndpointsID is the ID to look up the app runner service endpoint.
```
Comment (huanjani) [inline internal/pkg/aws/apprunner/apprunner.go]: nit:
```suggestion
	// EndpointsID is the ID to look up the App Runner service endpoint.
```

File: internal/pkg/aws/secretsmanager/secretsmanager.go [hunk 2]
```diff
@@ -83,6 +83,7 @@ func (s *SecretsManager) DeleteSecret(secretName string) error {
 	return nil
 }
 
+// DescribeSecretOutput is the output returned by DescribeSecret.
```
Comment (huanjani) [inline internal/pkg/aws/secretsmanager/secretsmanager.go:86]: 🤣

Comment (dannyrandall) [reply internal/pkg/aws/secretsmanager/secretsmanager.go:86]: i'm very creative

File: internal/pkg/ecs/errors.go [hunk 3]
```diff
@@ -24,6 +25,7 @@ func (e *ErrExitCode) Error() string {
 	return fmt.Sprintf("container %s in task %s exited with status code %d", e.containerName, e.taskId, e.exitCode)
 }
 
+// ExitCode returns the os exit code configured for this error.
```
Comment (huanjani) [inline internal/pkg/ecs/errors.go]: nit:
```suggestion
// ExitCode returns the OS exit code configured for this error.
```

File: internal/pkg/term/selector/selector.go [hunk 4]
```diff
@@ -434,7 +434,7 @@ func (s *TaskSelector) RunningTask(msg, help string, opts ...TaskOpts) (*awsecs.
 	return taskStrMap[task], nil
 }
 
-// GetDeployedServiceOpts sets up optional parameters for GetDeployedServiceOpts function.
+// GetDeployedWorkloadOpts sets up optional parameters for GetDeployedWorkloadOpts function.
```
Comment (huanjani) [inline internal/pkg/term/selector/selector.go:437]: wow, this one was ancient!

**Comment by huanjani** [comment]
Thanks so much for doing this! And for doing all that 🧹 !

**Review by huanjani — APPROVED** [review]

**Review by efekarakus — COMMENTED** [review]
This is great, ty so much Danny ❤️

**Pull request merged**